code/game/jobs/job_exp.dm

```diff
@@ -1,20 +1,22 @@
 
+// Defines
+
+#define EXP_TYPE_CREW			"Crew"
+#define EXP_TYPE_COMMAND		"Command"
+#define EXP_TYPE_ENGINEERING	"Engineering"
+#define EXP_TYPE_SECURITY		"Security"
+#define EXP_TYPE_SILICON		"Silicon"
+#define EXP_TYPE_SERVICE		"Service"
+#define EXP_TYPE_MEDICAL		"Medical"
+#define EXP_TYPE_SCIENCE		"Science"
+#define EXP_TYPE_SUPPLY			"Supply"
+#define EXP_TYPE_SPECIAL		"Special"
+#define EXP_TYPE_GHOST			"Ghost"
+#define EXP_TYPE_EXEMPT			"Exempt"
+
```

Comment:
These defines can't be here. They need to be in a file in the `__DEFINES` folder, or any code files that get read before this one won't be able to use them.